wsclient/src/main/resources/WSDLs/WsWorkunits-200.wsdl.notes

```diff
@@ -0,0 +1,6 @@
+WsWorkunits 2.00 required manual changes to WSDL to force successful compilation.

```

Comment:
WsWorkunits 2.00 doesn't exist on the ESP side right? Will there be a conflict if / when ESP moves to 2.00?